qa/L0_http_fuzz/test.sh

```diff
@@ -55,6 +55,47 @@ SERVER=/opt/tritonserver/bin/tritonserver
 SERVER_ARGS="--model-repository=$DATADIR"
 source ../common/util.sh
 
+# Remove this once foobuzz and tornado packages upgrade to work with python 3.10
+# This test tests the server's ability to handle poor input and not the compatibility 
+# with python 3.10. Python 3.8 is ok to use here.
+function_install_python38() {
+    apt-get update
+    apt-get remove -y python3
+    wget https://www.python.org/ftp/python/3.8.16/Python-3.8.16.tar.xz
+
+    # md5sum is not secure. Only use for sanity check
+    MD5SUM_PYTHON38=$(md5sum Python-3.8.16.tar.xz | awk '{ print $1 }' -)
+    CORRECT_MD5SUM_PYTHON38=621ac153586a3152e2ab7d3a8614df9a
+    if [ "$MD5SUM_PYTHON38" != "$CORRECT_MD5SUM_PYTHON38" ]; then
+        echo "md5sum of downloaded Python-3.8.16.tar.xz does not match! $MD5SUM_PYTHON38 != $CORRECT_MD5SUM_PYTHON38"
+        RET=1
+    fi 
+
+    # check the file size as well
+    FILE_SIZE_PYTHON38=$(ls -l Python-3.8.16.tar.xz | awk '{ print $5 }' -)
+    CORRECT_FILE_SIZE_PYTHON38=19046724
+    if [ "$FILE_SIZE_PYTHON38" != "$CORRECT_FILE_SIZE_PYTHON38" ]; then
+        echo "file size is not correct! $FILE_SIZE_PYTHON38 != $ $CORRECT_FILE_SIZE_PYTHON38" 
+        RET=1
+    fi
+    echo "Validated md5sum and file size of Python-3.8.16.tar.xz"
+
+    # Unpack pythonn and install 
+    tar -xf Python-3.8.16.tar.xz
+    cd Python-3.8.16
+    apt-get install -y libsqlite3-dev libffi-dev
+    ./configure --enable-loadable-sqlite-extensions
+    make 
+    make install
+
+    # Install test script dependencies
+    pip3 install --upgrade wheel setuptools boofuzz==0.3.0 numpy pillow attrdict future grpcio requests gsutil \
+                            awscli six grpcio-channelz prettytable virtualenv
+}
+WORKING_DIR=`pwd`
```

Comment:
Small suggestion - maybe this is `pushd`, `popd` in the function itself - keeping the function self contained with regards to directories.

Reply:
Just had to read up on this. Will use this in the future. With [@Tabrizian's suggestion](https://github.com/triton-inference-server/server/pull/5776#discussion_r1196787856) I don't think this will be needed. Will wait for the CI to finish though.